pool/connection_pool_test.go

```diff
@@ -130,6 +144,74 @@ func TestConnSuccessfully(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestConn_NoExecuteRole_Supported(t *testing.T) {
+	test_helpers.SkipIfWatchOnceUnsupported(t)
+
+	healthyServ := servers[0]
+
+	ctx, cancel := test_helpers.GetPoolConnectContext()
+	defer cancel()
+	connPool, err := pool.Connect(ctx,
+		[]pool.Instance{makeNoExecuteInstance(healthyServ, connOpts)})
+	require.Nilf(t, err, "failed to connect")
+	require.NotNilf(t, connPool, "conn is nil after Connect")
+
+	defer connPool.Close()
+
+	args := test_helpers.CheckStatusesArgs{
+		ConnPool:           connPool,
+		Mode:               pool.ANY,
+		Servers:            []string{healthyServ},
+		ExpectedPoolStatus: true,
+		ExpectedStatuses: map[string]bool{
+			healthyServ: true,
+		},
+	}
+
+	err = test_helpers.CheckPoolStatuses(args)
+	require.Nil(t, err)
```

Comment:
Please, send a `PingRequest` just to ensure that everything work fine.

Reply:
With `test` user + we send requests in other tests, so we need to check it. But with `testNoExec` we don't have another tests with requests, so let's ensure here.

Reply:
Updated.